lib/src/main/kotlin/com/swmansion/starknet/provider/gateway/GatewayProvider.kt

```diff
@@ -28,10 +29,10 @@ import kotlinx.serialization.json.*
  * @param httpService service used for making http requests
  */
 class GatewayProvider(
-    private val feederGatewayUrl: String,
-    private val gatewayUrl: String,
+    val feederGatewayUrl: String,
+    val gatewayUrl: String,
     override val chainId: StarknetChainId,
-    private val httpService: HttpService,
+    val httpService: HttpService,
```

Comment:
Are we sure we want to expose service here?

Reply:
I think service can remain private